autobot-backend/llc/services/reporting_line.py

```
@@ -0,0 +1,440 @@
+        if (ceo.type, ceo.id) != (current.type, current.id) and len(
+            managers
+        ) < max_depth:
+            managers.append(ceo)
+
+        # Owners sit above the CEO, all of them, as equals. They are appended
+        # together rather than picked between: there is no primary owner and
+        # none should be inferred.
+        if len(managers) < max_depth:
+            for owner in await self._owners(session, company_id):
+                if (owner.type, owner.id) != (current.type, current.id):
+                    managers.append(owner)
```

Ruff (0.16.3)

[warning] Use `list.extend` to create a transformed list

(PERF401)